loops-and-arrays.js

```diff
@@ -18,14 +18,11 @@ for (let i = 0; i < names.length; i++) {
 }
 
```

Comment:
Ze willen dat je de namen in de array vervangt met een koosnaam, in dit geval betekent dat het aanpassen van de array. Nu log je ze alleen met -je erachter, de array zelf wordt niet aangepast. Zie uitwerkingen als je er niet uitkomt